### ISSUE TITLE
I want to use haxophone with Raspberry Pi Zero 2 W 

### ISSUE THREAD

**Issue by hisayan** [cardonabits/haxo-hw#53]
I want to use haxophone with Raspberry Pi Zero 2 W (not Zero).

I tried using disk image for Zero.
https://bit.ly/haxo-rpiz
but It didn't work.

I tried using the disk image for Raspberry Pi 3.
https://bit.ly/haxo-rpi3
It worked well.

Is there any problem with this?

There may be others who have similar problems, so I'm here to let you know.

**Comment by jcard0na** [comment]
Hi Tomoyuki,

Thanks for letting us know!  I've had preparing an image for the Pi Zero 2 on my TODO list but have not found the time to do it yet.  I suspected that the RPi3 image would work based on the fact that they are both based on the same Cortex-A53 CPU, but never tested it.

And now you have 😄 Thank you!

> Is there any problem with this?

The only problem that you will have is that the image I released for the Pi3 [https://bit.ly/haxo-rpi3](https://bit.ly/haxo-rpi3) does not run the latest version of haxo-rs.  In particular you are missing USB Midi support and the startup chime that plays after boot.

If you are inclined to hacking, I recommend that you try to [build the latest version](https://github.com/cardonabits/haxo-hw/blob/main/docs/development.md#compilation-method-1-native-compilation-on-the-raspi) yourself.  Regardless of that,   I'll update the released image for the RPi3 (and now also Zero 2) after the holidays (I'm currently traveling with my Haxophone, but without a Pi Zero 2)

Thanks, and happy haxophoning!

cc: @pomalaterre
